x-pack/plugins/enterprise_search/server/routes/enterprise_search/search_applications.ts

```diff
@@ -215,31 +241,47 @@ export function registerSearchApplicationsRoutes({ log, router }: RouteDependenc
       validate: { params: schema.object({ engine_name: schema.string() }) },
     },
     elasticsearchErrorHandler(log, async (context, request, response) => {
-      try {
-        const { client } = (await context.core).elasticsearch;
+      const { client } = (await context.core).elasticsearch;
 
-        const engine = (await client.asCurrentUser.searchApplication.get({
+      let engine;
+      try {
+        engine = (await client.asCurrentUser.searchApplication.get({
           name: request.params.engine_name,
         })) as EnterpriseSearchApplication;
```

Comment:
nitpick: is this type cast still necessary?

Reply:
probably not since we are using the ES client now - I will remove these in a follow up when also renaming every occurrence of `engine_name` that we still have here.